.github/workflows/continuous-integration.yml

```diff
@@ -58,7 +58,7 @@ jobs:
             fpmodel: SP
     env:
       FC: gfortran-${{ matrix.gfortran-version }}
-      FFLAGS: "-m64 -std=f2008 -march=native -fbounds-check -fmodule-private -fimplicit-none -finit-real=nan"
```

Comment:
These flags are not supposed to change the result and should stay. If they change the result, we should understand why (compiler bug, library but, etc.).

However, it feels right to drop `-march=native` (and probably `-m64`). It would be nice to ensure that dropping `-std=f2008` does not affect the results as well.